code-of-conduct.md

```diff
@@ -12,21 +12,33 @@ We expect everyone involved in our community to follow this code of conduct. Thi
 
 We do not tolerate harassment of people at our events or space in any form. 
 
+
+We do not seek to list all cases of unacceptable behavior, but provide examples to help guide our community in thinking through how to respond when we experience these types of behavior, whether directed at ourselves or others. 
+
+Some guidelines on appropriate behavior:
+
+* If you are unsure if something is appropriate behavior, err on the side of caution. 
+* Each person you interact with can define what is appropriate for them. 
+* Impact matters more than intent. 
+* It is your responsibility to ensure that your behavior does not have a negative impact.
+* Problems happen when we assume that our way of thinking or behaving is the norm or ok with everyone. This is particularly problematic when we are in a position of power or privilege.
+
 Behavior considered as harassment includes, but is not limited to:
 
 1. Offensive comments including epithets, slurs, negative stereotyping, verbal kidding, teasing, jokes, or intimidating acts related to race, religion, gender, gender identity and expression, sexual orientation, disability, or physical appearance. Consider that calling attention to differences can feel alienating.
 2. Gratuitous sexual or obscene objects, images, discussions, or behavior.
-3. Physical contact, romantic advances, or sexual attention that continues after a request to stop or "no" response.
-4. Subtle or explicit demands for sexual activity.
-5. Verbal abuse.
-6. Threats or incitement of violence towards any individual, including encouraging a person to engage in self-harm
-7. Deliberate intimidation by words, gestures, body language, or menacing behavior
-8. Stalking
-9. Harassing photography or recording, including logging online activity for harassment purposes
-10. Continued one-on-one contact or communication after requests to cease
-11. Deliberate “outing” of a sensitive aspect of a person’s identity without their consent
-12. Deliberate misgendering. This includes dead-naming or persistently using a pronoun that does not correctly reflect a person’s gender identity
-13. Retaliating against anyone who files a complaint that someone has violated this code of conduct 
+3. Physical contact, romantic advances, or sexual attention that continues after a request to stop or a negative/"no" response.
```

Comment:
Or lack of affirmative "yes". These guidelines will always be written for people who are bad at social cues and need the rules to have as a reference guide.

Reply:
In this section I made minor readability edits to pts 3 and what-is-now-12-but-was-11. I also, to address the issue you're commenting on around affirmative consent (which was also brought up in those words in Tuesday's meeting), added point 4 (which moved points 4-13 into 5-14 in the diff, making it difficult to read on Github default settings)

My thought is, which I believe was mentioned in the public board meeting by someone else as well - the intent is not to prohibit a member being asked out by another member once in a respectful manner, and it's not easy to obtain "yes you can ask me on a date" until you have actually done the asking-out (as one example). Calling it "unwanted" is likewise not necessarily possible to judge in advance. So I wanted to break this up into two points, where all repeated behavior after being told to stop is prohibited (though we do call out touch and romantic/sexual behaviors as especially risky), and for the behaviors in that list where it is always possible to receive consent in advance, you have to receive consent. 

I also worded new-4 as "informed and voluntary" consent because I couldn't find a straightforward, short, and applicable definition of "affirmative consent" already written on a web search (I didn't look that thoroughly though), but I saw those words come up repeatedly in the definitions I did find. I do not think everyone has the same ideas in their heads of what affirmative consent means; I think informed and voluntary are more specific. Other adjective suggestions are welcome!

One additional thought on this is that obviously sexual activity is already banned at denhac in several other places in this document, so this would be aimed mostly at members interacting outside the space. I would like to have the latitude to address reports of a member violating our code of conduct outside the space via formal reporting procedures in egregious cases which impact space safety (though I don't think it is necessarily something we want to touch if it doesn't impact the safety of our space/of a significant part of our membership) - I am not sure if/how to clarify that, or even if it is something others agree with me on.